diff --ignore-blank-lines --ignore-space-change -u -r otp-22.0/lib/asn1/c_src/Makefile otp-22.0-patched/lib/asn1/c_src/Makefile
--- otp-22.0/lib/asn1/c_src/Makefile	2019-05-27 23:31:53.000000000 +0100
+++ otp-22.0-patched/lib/asn1/c_src/Makefile	2019-05-27 23:38:17.000000000 +0100
@@ -136,8 +136,5 @@
 release_spec: opt
 	$(INSTALL_DIR) "$(RELSYSDIR)/priv/lib"
 	$(INSTALL_PROGRAM) $(NIF_SHARED_OBJ_FILE) "$(RELSYSDIR)/priv/lib"
-	$(INSTALL_DIR) "$(RELSYSDIR)/c_src"
-	$(INSTALL_DATA) *.c "$(RELSYSDIR)/c_src"
 
 release_docs_spec:
-
diff --ignore-blank-lines --ignore-space-change -u -r otp-22.0/lib/crypto/c_src/Makefile.in otp-22.0-patched/lib/crypto/c_src/Makefile.in
--- otp-22.0/lib/crypto/c_src/Makefile.in	2019-05-27 23:31:53.000000000 +0100
+++ otp-22.0-patched/lib/crypto/c_src/Makefile.in	2019-05-27 23:39:11.000000000 +0100
@@ -223,16 +223,12 @@
 include $(ERL_TOP)/make/otp_release_targets.mk
 
 release_spec: opt
-	$(INSTALL_DIR) "$(RELSYSDIR)/priv/obj"
 	$(INSTALL_DIR) "$(RELSYSDIR)/priv/lib"
 	$(INSTALL_DATA) $(NIF_MAKEFILE) "$(RELSYSDIR)/priv/obj"
-	$(INSTALL_PROGRAM) $(CRYPTO_OBJS) "$(RELSYSDIR)/priv/obj"
 	$(INSTALL_PROGRAM) $(NIF_LIB) "$(RELSYSDIR)/priv/lib"
 ifeq ($(DYNAMIC_CRYPTO_LIB),yes)
-	$(INSTALL_PROGRAM) $(CALLBACK_OBJS) "$(RELSYSDIR)/priv/obj"
 	$(INSTALL_PROGRAM) $(CALLBACK_LIB) "$(RELSYSDIR)/priv/lib"
 endif
-	$(INSTALL_PROGRAM) $(TEST_ENGINE_OBJS) "$(RELSYSDIR)/priv/obj"
 	$(INSTALL_PROGRAM) $(TEST_ENGINE_LIB) "$(RELSYSDIR)/priv/lib"
 
 release_docs_spec:
diff --ignore-blank-lines --ignore-space-change -u -r otp-22.0/lib/erl_interface/src/Makefile.in otp-22.0-patched/lib/erl_interface/src/Makefile.in
--- otp-22.0/lib/erl_interface/src/Makefile.in	2019-05-27 23:31:58.000000000 +0100
+++ otp-22.0-patched/lib/erl_interface/src/Makefile.in	2019-05-27 23:39:39.000000000 +0100
@@ -827,15 +827,6 @@
 	$(INSTALL_DIR) "$(RELSYSDIR)/lib"
 	$(INSTALL_DIR) "$(RELSYSDIR)/bin"
 	$(INSTALL_DIR) "$(RELSYSDIR)/ebin"
-	$(INSTALL_DIR) "$(RELSYSDIR)/src/auxdir"
-	$(INSTALL_DIR) "$(RELSYSDIR)/src/connect"
-	$(INSTALL_DIR) "$(RELSYSDIR)/src/decode"
-	$(INSTALL_DIR) "$(RELSYSDIR)/src/encode"
-	$(INSTALL_DIR) "$(RELSYSDIR)/src/epmd"
-	$(INSTALL_DIR) "$(RELSYSDIR)/src/legacy"
-	$(INSTALL_DIR) "$(RELSYSDIR)/src/misc"
-	$(INSTALL_DIR) "$(RELSYSDIR)/src/prog"
-	$(INSTALL_DIR) "$(RELSYSDIR)/src/registry"
 	$(INSTALL_DIR) "$(RELEASE_PATH)/usr/include"
 	$(INSTALL_DIR) "$(RELEASE_PATH)/usr/lib"
 	$(INSTALL_DATA) $(APP_TARGET)  "$(RELSYSDIR)/ebin/$(APP_FILE)"
@@ -847,15 +838,6 @@
 ifneq ($(EXE_TARGETS),)
 	$(INSTALL_PROGRAM) $(EXE_TARGETS) "$(RELSYSDIR)/bin"
 endif
-	$(INSTALL_DATA) $(EXTRA)        "$(RELSYSDIR)/src"
-	$(INSTALL_DATA) connect/*.[ch]  "$(RELSYSDIR)/src/connect"
-	$(INSTALL_DATA) decode/*.[ch]   "$(RELSYSDIR)/src/decode"
-	$(INSTALL_DATA) encode/*.[ch]   "$(RELSYSDIR)/src/encode"
-	$(INSTALL_DATA) epmd/*.[ch]     "$(RELSYSDIR)/src/epmd"
-	$(INSTALL_DATA) misc/*.[ch]     "$(RELSYSDIR)/src/misc"
-	$(INSTALL_DATA) registry/*.[ch] "$(RELSYSDIR)/src/registry"
-	$(INSTALL_DATA) legacy/*.[ch]   "$(RELSYSDIR)/src/legacy"
-	$(INSTALL_DATA) prog/*.[ch]     "$(RELSYSDIR)/src/prog"
 
 release_docs:
 
diff --ignore-blank-lines --ignore-space-change -u -r otp-22.0/lib/os_mon/c_src/Makefile.in otp-22.0-patched/lib/os_mon/c_src/Makefile.in
--- otp-22.0/lib/os_mon/c_src/Makefile.in	2019-05-27 23:31:55.000000000 +0100
+++ otp-22.0-patched/lib/os_mon/c_src/Makefile.in	2019-05-27 23:41:39.000000000 +0100
@@ -127,7 +127,6 @@
 
 release_spec: opt
 	$(INSTALL_DIR) "$(RELSYSDIR)/src"
-	$(INSTALL_DATA) $(C_FILES) "$(RELSYSDIR)/src"
 	$(INSTALL_DIR) "$(RELSYSDIR)/priv/bin"
 	$(INSTALL_PROGRAM) $(TARGET_FILES) "$(RELSYSDIR)/priv/bin"
 
