diff -u -r otp-OTP-21.1.1.orig/lib/crypto/c_src/crypto.c otp-OTP-21.1.1/lib/crypto/c_src/crypto.c 
--- otp-OTP-21.1.1.orig/lib/crypto/c_src/crypto.c	2018-10-12 17:12:11.000000000 +0200
+++ otp-OTP-21.1.1/lib/crypto/c_src/crypto.c	2018-11-02 15:42:24.274260763 +0100
@@ -168,7 +168,8 @@
 #if OPENSSL_VERSION_NUMBER >= PACKED_OPENSSL_VERSION(0,9,8,'o') \
 	&& !defined(OPENSSL_NO_EC) \
 	&& !defined(OPENSSL_NO_ECDH) \
-	&& !defined(OPENSSL_NO_ECDSA)
+	&& !defined(OPENSSL_NO_ECDSA) \
+	&& !defined(OPENSSL_NO_EC2M)
 # define HAVE_EC
 #endif
 