--- otp-OTP-19.3.6.12.orig/lib/crypto/c_src/crypto.c	2018-11-07 15:56:33.000000000 +0100
+++ otp-OTP-19.3.6.12/lib/crypto/c_src/crypto.c	2018-11-07 16:01:02.000000000 +0100
@@ -102,7 +102,8 @@
 #if OPENSSL_VERSION_NUMBER >= OpenSSL_version(0,9,8,'o') \
 	&& !defined(OPENSSL_NO_EC) \
 	&& !defined(OPENSSL_NO_ECDH) \
-	&& !defined(OPENSSL_NO_ECDSA)
+	&& !defined(OPENSSL_NO_ECDSA) \
+    && !defined(OPENSSL_NO_EC2M)
 # define HAVE_EC
 #endif
 
