diff -B -u -r otp-OTP-22.0/lib/crypto/c_src/Makefile.in otp-OTP-22.0-patched/lib/crypto/c_src/Makefile.in
--- otp-OTP-22.0/lib/crypto/c_src/Makefile.in	2019-05-15 16:51:59.000000000 +0300
+++ otp-OTP-22.0-patched/lib/crypto/c_src/Makefile.in	2019-05-15 16:52:03.000000000 +0300
@@ -114,7 +114,7 @@
 DYNAMIC_CRYPTO_LIB=@SSL_DYNAMIC_ONLY@

 ifeq ($(DYNAMIC_CRYPTO_LIB),yes)
-SSL_DED_LD_RUNTIME_LIBRARY_PATH = @SSL_DED_LD_RUNTIME_LIBRARY_PATH@
+SSL_DED_LD_RUNTIME_LIBRARY_PATH =
 CRYPTO_LINK_LIB=$(SSL_DED_LD_RUNTIME_LIBRARY_PATH) -L$(SSL_LIBDIR) -l$(SSL_CRYPTO_LIBNAME)
 EXTRA_FLAGS = -DHAVE_DYNAMIC_CRYPTO_LIB
 else
diff -B -u -r otp-OTP-22.0/lib/crypto/priv/Makefile otp-OTP-22.0-patched/lib/crypto/priv/Makefile
--- otp-OTP-22.0/lib/crypto/priv/Makefile	2019-05-15 16:03:43.000000000 +0300
+++ otp-OTP-22.0-patched/lib/crypto/priv/Makefile	2019-05-15 16:03:41.000000000 +0300
@@ -61,7 +61,7 @@
 # ----------------------------------------------------

 $(SO_NIFLIB): $(OBJS)
-	$(SO_LD) $(SO_LDFLAGS) -L$(SO_SSL_LIBDIR) -Wl,-R$(SO_SSL_LIBDIR) \
+	$(SO_LD) $(SO_LDFLAGS) -L$(SO_SSL_LIBDIR) \
 	-o $@ $^ -lcrypto

 $(DLL_NIFLIB): $(OBJS)
