diff -u -r otp_src_18.2-a/lib/crypto/c_src/Makefile.in otp_src_18.2/lib/crypto/c_src/Makefile.in
--- otp_src_18.2-a/lib/crypto/c_src/Makefile.in	2014-12-09 20:11:07.000000000 +0000
+++ otp_src_18.2/lib/crypto/c_src/Makefile.in	2015-01-14 12:45:52.181287995 +0000
@@ -89,7 +89,7 @@
 DYNAMIC_CRYPTO_LIB=@SSL_DYNAMIC_ONLY@
 
 ifeq ($(DYNAMIC_CRYPTO_LIB),yes)
-SSL_DED_LD_RUNTIME_LIBRARY_PATH = @SSL_DED_LD_RUNTIME_LIBRARY_PATH@
+SSL_DED_LD_RUNTIME_LIBRARY_PATH =
 CRYPTO_LINK_LIB=$(SSL_DED_LD_RUNTIME_LIBRARY_PATH) -L$(SSL_LIBDIR) -l$(SSL_CRYPTO_LIBNAME)
 EXTRA_FLAGS = -DHAVE_DYNAMIC_CRYPTO_LIB
 else
diff -u -r otp_src_18.2-a/lib/crypto/priv/Makefile otp_src_18.2/lib/crypto/priv/Makefile
--- otp_src_18.2-a/lib/crypto/priv/Makefile	2014-12-09 20:11:07.000000000 +0000
+++ otp_src_18.2/lib/crypto/priv/Makefile	2015-01-14 12:45:52.181287995 +0000
@@ -60,7 +60,7 @@
 # ----------------------------------------------------
 
 $(SO_NIFLIB): $(OBJS)
-	$(SO_LD) $(SO_LDFLAGS) -L$(SO_SSL_LIBDIR) -Wl,-R$(SO_SSL_LIBDIR) \
+	$(SO_LD) $(SO_LDFLAGS) -L$(SO_SSL_LIBDIR) \
 	-o $@ $^ -lcrypto
 
 $(DLL_NIFLIB): $(OBJS)
