diff -u -r otp_src_18.2-a/erts/etc/common/Makefile.in otp_src_18.2/erts/etc/common/Makefile.in
--- otp_src_18.2-a/erts/etc/common/Makefile.in	2014-12-09 20:11:07.000000000 +0000
+++ otp_src_18.2/erts/etc/common/Makefile.in	2015-01-14 12:44:58.017995780 +0000
@@ -550,10 +550,6 @@
 ifneq ($(INSTALL_TOP_BIN),)
 	$(INSTALL_PROGRAM) $(INSTALL_TOP_BIN) "$(RELEASE_PATH)"
 endif
-ifneq ($(INSTALL_MISC),)
-	$(INSTALL_DIR) "$(RELEASE_PATH)/misc"
-	$(INSTALL_SCRIPT) $(INSTALL_MISC) "$(RELEASE_PATH)/misc"
-endif
 ifneq ($(INSTALL_SRC),)
 	$(INSTALL_DIR) "$(RELEASE_PATH)/erts-$(VSN)/src"
 	$(INSTALL_DATA) $(INSTALL_SRC) "$(RELEASE_PATH)/erts-$(VSN)/src"
diff -u -r otp_src_18.2-a/erts/etc/unix/Install.src otp_src_18.2/erts/etc/unix/Install.src
--- otp_src_18.2-a/erts/etc/unix/Install.src	2014-12-09 20:11:07.000000000 +0000
+++ otp_src_18.2/erts/etc/unix/Install.src	2015-01-14 12:44:58.945983653 +0000
@@ -140,14 +140,5 @@
 cp -p ../releases/%I_SYSTEM_VSN%/no_dot_erlang.boot .
 cp -p $Name.boot start.boot
 cp -p ../releases/%I_SYSTEM_VSN%/$Name.script start.script
-#
-# Fixing the man pages
-#
-
-if [ -d "$ERL_ROOT/man" ]
-then
-    cd "$ERL_ROOT"
-    ./misc/format_man_pages "$ERL_ROOT"
-fi
 
 exit 0
